devdoc/CodeReviews.md

```diff
@@ -0,0 +1,55 @@
+Some general guidelines for code reviews.
+
+Code reviews have a few different goals:
+* Catch architectural or design problems
```

Comment:
While code reviews should catch these, we should aim to have caught them before any change ever reaches code-review stage.

Reply:
will add an extra section on PRs